docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -843,7 +843,21 @@ So if you need to call methods such as `verify` you should hang on to the mock i
 ==== Further simplification with `@InjectMock`
 
 Building on the features provided by `QuarkusMock`, Quarkus also allows users to effortlessly take advantage of link:https://site.mockito.org/[Mockito] for mocking the beans supported by `QuarkusMock`.
-This functionality is available with the `@io.quarkus.test.InjectMock` annotation if the `quarkus-junit5-mockito` dependency is present.
+
+[IMPORTANT]
+====
+This functionality is available with the `@io.quarkus.test.InjectMock` annotation **only if** the `quarkus-junit5-mockito` dependency is present:
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-junit5-mockito</artifactId>
+    <version>${quarkus.platform.version}</version>
```

Comment:
Could you drop the version? It should come with the BOM (and if not it's a bug we need to fix!).

Reply:
There you go!